policy/agent.go

```diff
@@ -68,6 +68,7 @@ func (o *Agent) GetBackendName() string {
 // overwrite the result status or any of the values in the result trust vector.
 func (o *Agent) Evaluate(
 	ctx context.Context,
+	appraisalContext map[string]interface{},
```

Comment:
Could there be a better name as the name collides with AppraisalContext on the proto interface which means completely different data structure!